Packs/CommonScripts/Scripts/ExtractIndicatorsFromTextFile/ExtractIndicatorsFromTextFile.py

```diff
@@ -1,6 +1,35 @@
 import demistomock as demisto
 from CommonServerPython import *
 from CommonServerUserPython import *
+import json
+
+
+def string_to_markdown(indicators: str) -> str:
```

Comment:
Why do we need this function?
tableToMarkdown won't do the trick?

Reply:
I tried, it didnt work for me :(
Maybe I've done it wrong?

Reply:
Its, not a table though, its a list so even if it works it will end up looking bad.

Reply:
@Shellyber